### ISSUE TITLE
Remote button too large

### ISSUE THREAD

**Issue by postblue** [hannesmannerheim/qvitter#303]
It depends on the traduction chars.

![](http://i.imgur.com/PWjiKsB.png)

**Comment by hannesmannerheim** [comment]
yes, that's the intended layout if the translation is too long. maybe you can shorten the french translation? "suivre fédéré"?
